go/logic/gtidset.go

```diff
@@ -0,0 +1,56 @@
+package logic
```

Comment:
With this being a new file, I didn't know if this needed the copyright statement at the top or not... dunno if that's necessary.

Reply:
If it's a new file, you get to pick the copyright statement, as long as it is compliant with this project's license. Best advice: MIT under your name.

Reply:
Yeah this was going to stay MIT. However after looking at your orchestrator code, I wonder if I should just use `gtid_subtract` instead.